datacenter/ucp/2.2/guides/access-control/deploy-stack-to-collection.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Deploy stack resources to a collection 
+description: Learn how to manage user access to stack resources by using collections. 
+keywords: UCP, authentication, user management, stack, collection, role
+---
+
+A stack file is the definition of an application, and in the stack file,
```

Comment:
I know that people informally talk about stack files and compose files, but officially there's just one concept: Compose files. I know that it's weird but the tools themselves are fairly consistent. `docker stack deploy --compose-file ...` and `compose.yml` in the UCP UI.

So we should use "Compose file" everywhere and make reduce the number of concepts new users need to learn.

Reply:
I also think that you kick off this article from a weird viewpoint and that makes it harder to drive the point home. What about structuring the intro/article like this:

1. UCP enforces role-based access control when you're deploying services
2. By default, you don't need to do anything. UCP will deploy your services to a default collection, which you can customize in your UCP profile page. Learn more about collections here.
3. Under the hood the collection where a service is deployed to is defined by a label
4. Here's an example of a `docker service create` command using that label so that you can see it working on a well-contained example without much complexity
5. You can also do this for your Compose files. Here's an example of a compose file using labels.